environment.yml

```diff
@@ -106,6 +106,7 @@ dependencies:
   - cftime  # Needed for downstream xarray.CFTimeIndex test
   - pyreadstat  # pandas.read_spss
   - tabulate>=0.8.3  # DataFrame.to_markdown
+  - natsort>=7.0.1 # DataFrame.sort_values
```

Comment:
is there a reason to require this recent version? (not a big deal but usually we are pretty loose esp since this is just for a doc example)

Reply:
Good point, removed the version.